data/themes/darktable.css

```diff
@@ -870,9 +886,9 @@ menuitem check
   margin-right: 5px;
 }
 
-/* Header and footer combobox */
-#header-toolbar combobox,
-#footer-toolbar combobox
+/* Header and footer labels */
+#header-toolbar label,
+#footer-toolbar label
 {
   margin: 0 0.5em;
 }
```

Comment:
Do not remove combobox here, add it between toolbar and label for both header and footer line. That will avoid having that setting applied for some label we have outside a combobox.